index.html

```diff
@@ -0,0 +1,153 @@
+        <a href="./index.html"><img class="summit-logo" src="./images/DGEELOGO1.png"  alt="DGEESlogo"/></a>
+        </div>
+
+        <nav class="navigation">
+          <a href="./about.html">About</a>
+          <a href="./index.html#program">Program</a>
+          <a href="#speakers-section">Speakers</a>
+          <a href="#sponsor">Sponsor</a>
+          <button class="dgees-btn" type="button">D'GEES Campaign</button>
+        </nav>
+      </div>
+    </header>
+
+    <div class="mobile-menu" id="mobile-menu">
+      <div class="mobile-menu-header">
+        <button type="button" class="close-button">&times;</button>
+      </div>
+      <nav>
+        <a href="./index.html" class="navigate"> Home </a>
+        <a href="./about.html" class="navigate"> About </a>
+        <a href="./index.html#program" class="navigate"> Program </a>
+        <a href="#speakers-section" class="navigate"> Speakers </a>
+        <a href="./about.html#sponsor" class="navigate"> Sponsor </a>
+      </nav>
+    </div>
+
+    <main class="main">
+      <div class="box">
+        <button type="button" class="hamburger">
+          <img src="./images/menu.png" alt="menu icon" />
+        </button>
+        <p class="hello">"Hello! INFOTECH WORLD"</p>
+        <h1 class="heading">D'GEES INFOTECH <br />GLOBAL SYSTEMS SUMMIT<br />2022
+        </h1>
+        <section class="begin">
+          <h2 class="intro">
+           D'GEES Infotech Global Summits are free events that bring the cloud computing community together to connect, collaborate, and learn about D'GEES. 
+          </h2>
+          <p class="due-date">2022.07.11(MON) ~ 2022.07.15(FRI)</p>
+          <p class="venue">@ E-library Multipurpose Hall, Uyo</p>
+        </section>
+      </div>
+
+      <!-- The main program section -->
+      <section class="program" id="program">
+        <div class="prog">
+          <h3 class="main-program">Main Program</h3>
+        </div>
+        <ul class="activities">
+          <li class="activity">
+            <img class="img-activity" src="./images/lectureicon.jpg" alt="lecture icon"/>
+            <p class="description">Lectures</p>
+            <p class="explanation">Come and listen to D'GEES InfoTech Global leaders, experts, partners, and customers on how to drive digital transformation using AI
+            </p>
+          </li>
+
+          <li class="activity">
+          <img class="img-activity" src="./images/Exhibitionicon.jpg" alt="exhi icon"/>
+            <p class="description">D'GEES Exhibition</p>
+            <p class="explanation">Showcasing the global technological advancements & innovations.</p>
+          </li>
+          
+          <li class="activity">
+            <img class="img-activity" src="./images/workshopicon.jpg" alt="works icon"/>
+            <p class="description">Workshop</p>
+            <p class="explanation">The event provides a platform for accelerating investment-ready start-ups.
+            </p>
+          </li>
```

Comment:
The workshop program image does not display properly. Kindly look into this to make sure that all your images are visible on page load.

![image](https://user-images.githubusercontent.com/38088818/176725144-eb461143-3540-4933-9de2-2348b72c1c56.png)